lib/src/source/hosted.dart

```diff
@@ -883,6 +886,12 @@ class HostedSource extends CachedSource {
     });
   }
 
+  http.Request _createArchiveRequest(Uri url) {
+    var request = http.Request('GET', url);
+    request.headers[io.HttpHeaders.cacheControlHeader] = 'no-transform';
```

Comment:
Could you write a comment about what this does and why we need it (are we sure we need it - is this not mainly relevant to images, I would expect the checksum that gcs serves to be reliable)?

Reply:
So, I added that due to some guidance in the Decompressive transcoding section on [Transcoding of gzip-compressed files](https://cloud.google.com/storage/docs/transcoding#decompressive_transcoding):

> Preventing decompressive transcoding is useful, for example, if you want to reduce egress cost or time or if you want to validate the downloaded objects have the expected crc32c/md5 checksums.

I'm not familiar enough with the storage infrastructure to know if it's strictly needed for our case.

In any case, I should comment that and I will shortly.